mllib/src/test/scala/org/apache/spark/ml/feature/Word2VecSuite.scala

```diff
@@ -144,6 +144,31 @@ class Word2VecSuite extends SparkFunSuite with MLlibTestSparkContext with Defaul
     }
   }
 
+  test("findSynonymsArray") {
```

Comment:
Once you combine them, you can check findSynonymsArray by comparing it against findSynonyms.collect